industrial_moveit_benchmarking/src/stomp_valgrind.cpp

```diff
@@ -152,6 +152,7 @@ int main (int argc, char *argv[])
   }
   t2 = ros::Time::now();
 
-  ROS_ERROR("Average time spent calculating trajectory: %4.10f seconds", (t2-t1).toSec()/test_runs);
+  std::string summary = boost::str(boost::format("Average time spent calculating trajectory: %4.10f seconds") % ((t2-t1).toSec()/test_runs));
+  std::cout<<summary<<std::endl;
```

Comment:
This seems like an unrelated change?

Reply:
The fact that it printed an error seemed confusing to me as it wasn't really an error, just a workaround to force the message to print on the console and so I changed it to print the result as a regular message.